DataManagementSystem/Client/FTSJob.py

```diff
@@ -35,14 +35,14 @@
 from DIRAC.Resources.Catalog.FileCatalog     import FileCatalog
 from DIRAC.Core.Utilities.ReturnValues import returnSingleResult
 
-import fts3.rest.client.easy as fts3
+import fts3.rest.client.easy as fts3 #pylint: disable=import-error
```

Comment:
Because there's no `pip install fts3` so this is shown as an error by pylint tests :-/